src/main/java/org/commcare/formplayer/session/MenuSession.java

```diff
@@ -442,11 +476,15 @@ public SerializableMenuSession serialize() {
         return session;
     }
 
+    public ArrayList<PersistentCommand> getPersistentMenu() {
+        return persistentMenuHelper.getPersistentMenu();
```

Comment:
```suggestion
    public ArrayList<PersistentCommand> getPersistentMenu() {
        return persistentMenuHelper.getPersistentMenu();
    }
```

Looks like an unmatched `{` causing a build failure:
```
/home/runner/work/formplayer/formplayer/src/main/java/org/commcare/formplayer/session/MenuSession.java:482: error: illegal start of expression
    public void setWindowWidth(String windowWidth) {
    ^
```

Reply:
Thanks for flagging Ethan, [corrected](https://github.com/dimagi/formplayer/pull/1594/commits/711ead5c11dd4506d0ee11f1aabdc603b724c75b)